apps/tx-builder/src/hooks/useServices/interfaceRepository.ts

```diff
@@ -76,7 +75,7 @@ class InterfaceRepository {
       .map((m: any) => {
         return {
           inputs: m.inputs || [],
-          name: m.name || (m.type === 'fallback' ? 'fallback' : 'receive'),
+          name: m.name || (m.type === 'fallback' ? 'fallback' : m.type === 'constructor' ? 'constructor' : 'receive'),
           payable: this._isMethodPayable(m),
         };
       });
```

Comment:
Line 83 uses `fallback` to check for a `payableFallback` is this property still used? If so it should include `receive`, right?